pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -277,7 +320,7 @@ func (mattermost *ClusterInstallation) GenerateDeployment(dbUser, dbPassword str
 							Ports: []corev1.ContainerPort{
 								{
 									ContainerPort: 8065,
-									Name:          mattermost.Name,
+									Name:          "app",
```

Comment:
Is there a reason we changed this value away from being based on the cluster installation name?

Reply:
this name is just a name and it is limited to 15 chars, or we truncate the name or we hardcode like this

Reply:
Okay, this is fine for now.